src/engine/sys/manifests/dpi-aware.manifest

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">
+  <application xmlns="urn:schemas-microsoft-com:asm.v3">
+    <windowsSettings>
+      <!-- Specifies whether the current process is dots per inch (dpi) aware (old).
+           See https://docs.microsoft.com/en-us/windows/win32/sbscs/application-manifests#dpiaware -->
+      <dpiAware xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">true/PM</dpiAware>
+      <!-- Specifies whether the current process is dots per inch (dpi) aware (new).
+           See https://docs.microsoft.com/en-us/windows/win32/sbscs/application-manifests#dpiawareness -->
+      <dpiAwareness xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">PerMonitorV2, PerMonitor</dpiAwareness>
```

Comment:
Application is Per-Monitor DPI Aware v.2

![image](https://user-images.githubusercontent.com/1983411/116013377-939f4480-a638-11eb-8252-b5177e83e2f1.png)

Reply:
Oh that's good!